DEVELOPMENT.md

```diff
@@ -226,7 +226,9 @@ make debug-cache
 
 ## Publishing
 
-To publish new versions of the docker image for use in a deployed environment, set up a docker buildx builder (for being able to build the image to run on different cpu architectures)
+Automatic publishing and deploying of new versions of the proxy service for Kava Labs operated infrastructure follows [this process](https://kava-labs.atlassian.net/wiki/spaces/ENG/pages/1235320861/Deploying+New+Versions+of+the+Proxy+Service)
```

Comment:
@drklee3 does this address the ambiguity around deployment flows(discussed during our last retro)? If not whats the blurb we need to put in to fill in the missing documentation?

Reply:
Yup that looks good!